plugins/flusher/http/flusher_http.go

```diff
@@ -281,10 +282,17 @@ func (f *FlusherHTTP) flush(data []byte, varValues map[string]string) (ok, retry
 		req.Header.Add(k, v)
 	}
 	response, err := f.client.Do(req)
-	logger.Debugf(f.context.GetRuntimeContext(), "request [method]: %v; [header]: %v; [url]: %v; [body]: %v", req.Method, req.Header, req.URL, string(data))
+	if logger.DebugFlag() {
```

Comment:
Why add DebugFlag?
Because string(data) is slow?

Reply:
yes